src/MSALWrapper.Test/AuthFlow/AuthFlowExecutorTest.cs

```diff
@@ -616,6 +616,29 @@ public async Task HasThreeAuthFlows_Returns_Early_With_TokenResultAndErrors()
             result.Errors.Should().BeEquivalentTo(new[] { errors1[0], errors2[0], errors2[1] });
         }
 
+        [Test]
+        public async Task HasNullResultAuthFlow_Returns_With_BadImplementationException()

```

Comment:
This method needs renaming according to the new Exception.

Reply:
Thanks. I removed this unit test because this unit test has been covered by [SingleAuthFlow_Returns_Null_AuthFlowResult](https://github.com/AzureAD/microsoft-authentication-cli/blob/9128e9de96aa2840843fb80daadab9c0978ad724/src/MSALWrapper.Test/AuthFlow/AuthFlowExecutorTest.cs#L162).

Reply:
oops! got it!